src/main/java/io/github/bonigarcia/wdm/UrlFilter.java

```diff
@@ -45,6 +45,7 @@
     final Logger log = getLogger(lookup().lookupClass());
 
     public List<URL> filterByOs(List<URL> list, String osName) {
+        System.out.println("Filtering drivers by OS: " + osName);
```

Comment:
Do not use System.out.println for logging. Use log instead (in fact it was already used, at trace level).